judo-runtime-core-validator/src/main/java/hu/blackbelt/judo/runtime/core/validator/UniqueAttributeValidator.java

```diff
@@ -91,22 +91,21 @@ public Collection<ValidationResult> validateValue(Payload instance, final EStruc
 
         final Collection<ValidationResult> validationResults = new ArrayList<>();
 
-        if (!(feature instanceof EAttribute)
+        if (!(feature instanceof EAttribute attribute)
                 || !asmUtils.getMappedAttribute((EAttribute) feature).isPresent()
```

Comment:
There is no change in the logic and it is not better to read. So what is the reason?

Reply:
[27ceff5](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/140/commits/27ceff5a98c30e4d60020d6068861942123033ac)